aws_pricing/datadog_checks/aws_pricing/data/conf.yaml.example

```diff
@@ -0,0 +1,9 @@
+init_config:
+
+instances:
+    ## @param AmazonEC2 - string - optional
+    ## Rate codes to monitor for AmazonEC2 service.
+    #
+  - AmazonEC2: rate_code_1,rate_code_2,rate_code_3
+    # - AmazonCloudFront: rate_code_1,rate_code_2,rate_code_3
```

Comment:
I think this can be removed, since we want to give only one example of instance definition.